asmcomp/arm64/emit.mlp

```diff
@@ -78,12 +78,15 @@ let prologue_required = ref false
 
 let contains_calls = ref false
 
-let frame_size () =
-  let sz =
-    !stack_offset +
+let initial_frame_size () =
```

Comment:
This is not really a frame size because it is not aligned, right? Could you find some other name for it?

Reply:
Right. How about initial_stack_offset?